test/integration/server.test.js

```diff
@@ -76,6 +76,16 @@ test('POST /query', async (t) => {
     console.log(response.payload);
 });
 
+test('GET /getWorkflowList', async (t) => {
+    const response = await fastify.inject({
+        method: 'GET',
+        url: '/getWorkflowList'
+    });
+    assert.strictEqual(response.statusCode, 200);
+    const payloadList = response.payload;
+    console.log("Workflows: ", payloadList);

```

Comment:
I'm curious if this payloadList is actually a list of workflows -- I'm assuming you tested locally?